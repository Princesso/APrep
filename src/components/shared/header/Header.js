import React from 'react';
import styled from 'styled-components'
import colors from '../../../styles/colors'
import './header.css'

const Header = () => {
  return(
    <div className="header">
      <div > Send-It</div>
      <div>
        <ul>
        <li>Home</li>
        <li>About</li>
        <li>Login</li>
        </ul>
      </div>
    </div>
  )
}

export default Header