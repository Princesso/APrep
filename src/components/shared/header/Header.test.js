import React from 'react';
import Header from './Header';
import renderer from 'react-test-renderer';
import {Provider} from 'react-redux';
import Store from '../../../store';
import { BrowserRouter, Route, Switch } from 'react-router-dom'

test('Registration Form', () => {
  const component = renderer.create(
    <Provider store={Store}>
      <BrowserRouter>
          <Switch>
            <Header/>
          </Switch>
      </BrowserRouter>
    </Provider>
  )

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})