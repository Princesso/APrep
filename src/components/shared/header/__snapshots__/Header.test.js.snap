// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Registration Form 1`] = `
<div
  className="header"
>
  <div
    className="logo"
  >
     Send-It
  </div>
  <div>
    <ul>
      <li>
        <a
          className="link"
          href="/"
          onClick={[Function]}
        >
          HOME
        </a>
      </li>
      <li>
        <a
          className="link"
          href="/"
          onClick={[Function]}
        >
          ABOUT
        </a>
      </li>
      <li>
        <a
          className="link"
          href="/login"
          onClick={[Function]}
        >
          LOGIN
        </a>
      </li>
    </ul>
  </div>
</div>
`;
